polaris-patterns/src/configure.ts

```diff
@@ -14,3 +16,34 @@ if (typeof window !== 'undefined') {
 }
 
 export const polarisPatternsVersion = '{{POLARIS_PATTERNS_VERSION}}';
+
+export const DEFAULT_I18N_DETAILS: I18nDetails = {
+  locale: 'en',
+  country: 'CA',
+  currency: 'USD',
+  timezone: 'EST',
```

Comment:
I'm not an expert in timezones (yet), but I think that using civil timezone identifiers is preferred:

```suggestion
  timezone: 'America/New_York',
```

Reply:
I looked a little bit into this, what do you think of leaving the default `timezone` unset? From what I can tell, I don't think that `react-i18n` requires it (unlike `country`, which is required if you want to use `i18n.weekStartDay()`). 

These are the docs on `timeZone` for [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) which suggests it will automatically select a time zone from the javascript run time if it's not provided:
> The time zone to use. The only value implementations must recognize is "UTC"; the default is the runtime's default time zone. Implementations may also recognize the time zone names of the [IANA time zone database](https://www.iana.org/time-zones), such as "Asia/Shanghai", "Asia/Kolkata", "America/New_York".